rpcs3/Emu/Io/LogitechG27.h

```diff
@@ -111,7 +120,8 @@ class usb_device_logitech_g27 : public usb_device_emulated
 	std::mutex m_sdl_handles_mutex;
 	SDL_Joystick* m_led_joystick_handle = nullptr;
 	SDL_Haptic* m_haptic_handle = nullptr;
-	std::map<u32, std::vector<SDL_Joystick*>> m_joysticks;
+	std::map<u64, std::vector<SDL_Joystick*>> m_joysticks;
+	std::map<u64, std::vector<SDL_Joystick*>> m_joysticks_v1;
```

Comment:
I don't think this has been a feature long enough to warrant adding spaghetti code for versioning yet.